### ISSUE TITLE
doesn't seem to work with Github Enterprise?

### ISSUE THREAD

**Issue by matthewfischer** [peter-evans/rebase#139]
Using Github Enterprise (GHE) the URLs for the repos and APIs are different. I have set GITHUB_SERVER_URL, GITHUB_GRAPHQL_URL and GITHUB_API_URL without success.  It seems to be ignoring the GRAPHQL URL variable entirely, or I am setting it wrong.

`Error: request to https://api.github.com/graphql failed.`

```name: Rebase On Schedule
on:
  workflow_dispatch:

jobs:
  rebase:
    strategy:
      matrix:
        repo: [foo/bar-repo]
    runs-on: [ build_system ]
    steps:
      - uses: peter-evans/rebase@v2
        env:
          GITHUB_SERVER_URL: https://github.xyz.com
          GITHUB_API_URL: https://github.xyz.com/api/v3
          GITHUB_GRAPHQL_URL: https://github.xyz.com/api/graphql
        with:
          token: ${{ secrets.PAT_TOKEN }}
          repository: ${{ matrix.repo }}
```

I am not quite sure where it's failing (is there a debug mode?) specifically.

**Comment by peter-evans** [comment]
Hi @matthewfischer 

I've released a new version of the action (`v2.1.0` / `v2`), which hopefully works on GHE. It's difficult for me to test so please let me know if you run into any further issues.

By the way, I'm not sure if you need to set the environment variables explicitly in the workflow like that. Try without first, because I have a feeling those are set automatically on the runner.

**Comment by matthewfischer** [comment]
Done and it still works. Thanks again!
